### DISCUSSION TITLE
Using vars and lets?

### DISCUSSION THREAD

**Discussion by paperboy6891** [paperboy6891/Final-Project#5] [Q&A] [answered]
how would i know where to use let and where to use vars

**Comment by arhmAli** [comment, marked as answer]
Well the answer is a bit big but to sum up you should try to use let for values that need or can be reassigned (basically variable) and const for values that do not need to change.
in my personal opinion var shouldn't be used in any case although it was used for global declaration but its use now is limited to almost none!
Hope this helped

**Reply by paperboy6891** [reply]
Ok got it
